src/css/components/_step-by-step.scss

```diff
@@ -0,0 +1,47 @@
+.step-by-step {
+  display: flex;
+  gap: var(--space--base);
+  position: relative;
+  text-align: center;
+
+  &::before {
+    background-color: var(--color--text);
+    content: "";
+    display: block;
+    height: var(--border-width--thick);
+    left: 0;
+    position: absolute;
+    top: 50%;
+    width: 100%;
+    z-index: 0;
+  }
+}
+
+.step-by-step__item {
+  align-items: center;
+  background-color: var(--color--background-primary);
+  border: var(--border--base);
+  border-radius: var(--border-radius--base);
+  display: flex;
+  flex-direction: column;
+  padding: var(--space--base);
+  z-index: 1;
+  //width: clamp(9rem, 14vw, 15rem);
+  width: 15rem;
```

Comment:
I feel like instead of a set width we can use relative units here. Maybe `flex: 1;`. The one issue with setting a width is that it creates a horizontal scroll on smaller viewports.

Reply:
I updated and removed the fixed width and replaced it with the flex stuff like you suggested and it should be good to go (hopefully). :)